Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -14,7 +14,9 @@
 //
 //******************************************************************************
 
+#include <COMIncludes.h>
```

Comment:
Let us move these into ApplicationMain.h so other files that include this header do not have to bother guarding these explictly